docs/README.md

```diff
@@ -0,0 +1,9 @@
+<!-- Front page of web site -->
+
+# Web page test
+
+This should be the main page of the Github Pages site.
```

Comment:
Was this meant to be left in?

Reply:
The web page is just a stub at the moment. I need to spend some quality time with my text editor creating some content.